pipelines/postprocess_state_forecast.R

```diff
@@ -271,12 +271,15 @@ p <- arg_parser("Generate forecast figures") |>
 argv <- parse_args(p)
 model_run_dir <- path(argv$model_run_dir)
 
-base_dir <- path_dir(model_run_dir)
-
-disease_name_raw <- base_dir |>
-  path_file() |>
+# replace this with functionality from hewr
+disease_name_raw <- model_run_dir |>
+  path_split() |>
+  pluck(1) |>
+  tail(3) |>
+  head(1) |>
   str_extract("^.+(?=_r_)")
```

Comment:
This can happen now that #141 has landed. Use `hewr::parse_model_run_dir_path()`

Reply:
Resolved in https://github.com/CDCgov/pyrenew-hew/pull/136/commits/3f8139b920dbba75c425e6851be835b47d22f050